<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmGaugeCar</class>
 <widget class="QWidget" name="frmGaugeCar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="GaugeCar" name="gaugeCar1"/>
   </item>
   <item row="0" column="1">
    <widget class="GaugeCar" name="gaugeCar2"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSlider" name="horizontalSlider">
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GaugeCar</class>
   <extends>QWidget</extends>
   <header>gaugecar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
